src/crypter.js

```diff
@@ -7,9 +7,11 @@ const toUrlSafeBase64 = (input: Buffer) => input
   .toString('base64')
   .replace(/\+/g, "-")
   .replace(/\//g, "_")
-  .replace(/=*$/, m => m.length.toString());
+  .replace(/=*$/, m => m.length.toString()); // why not just strip everything?
 module.exports.toUrlSafeBase64 = toUrlSafeBase64;
 
+// I don't think this is needed -- node supports creating from a URL & Filename safe base64 string directly
+// -- BUT -- not sure what version that's from either so maybe needed for 4.x support?
```

Comment:
It does? Do you have a link to a function? I couldn't find anything. That aside this should probably be moved out of this class.

Reply:
Just in the regular docs for Buffer. It *should* also take care of the equal sign padding for you...

You should just be able to pass `Buffer.from(input, 'base64')` and have it work provided you're url encoding conforms to the spec.